components/builder-protocol/protocols/originsrv.proto

```diff
@@ -341,7 +341,7 @@ message OriginProject {
   optional uint64 owner_id = 7;
   optional string vcs_type = 8;
   optional string vcs_data = 9;
-  optional uint64 vcs_installation_id = 12;
+  optional uint32 vcs_installation_id = 12;
```

Comment:
I think a 32 bit is plenty but curious why the change here since it seems our convention is 64.

Reply:
Our convention is 64 but their API is sending 32

Reply:
Yeah I think that was my fault. Thats the type I used in our frank fork. Should be fine but in retrospect I'd have used u64.